### PR TITLE
Fix mistakes in register example

### PR DESCRIPTION
**Pull request by danielballan**
Discovered while @genematx was trying to actually use this.

### CODE REVIEW

File: docs/source/how-to/register.md [hunk 1]
```diff
@@ -76,8 +76,8 @@ Use the Python client, as in this example.
 
 ```py
 from tiled.client import from_uri
-from tiled.client.data_source import Asset, DataSource
 from tiled.structures.core import StructureFamily
+from tiled.structures.data_source import Asset, DataSource
```
Comment (padraic-shafer) [inline docs/source/how-to/register.md:80]:
```suggestion
from tiled.structures.data_source import Asset, DataSource, Management
```

File: docs/source/how-to/register.md [hunk 2]
```diff
@@ -87,11 +87,17 @@ client = from_uri("http://localhost:8000", api_key="...")
...
-    ),
+    data_sources=[
+        DataSource(
+        assets=[
+            Asset(data_uri="file:///...", num=1),
+            Asset(data_uri="file:///...", num=2)
+        ],
+        mimetype="multipart/related;type=image/tiff",
+        structure_family=StructureFamily.array,
+        management="external",
```
[the remark this answers is not in the document]
Comment (padraic-shafer) [reply docs/source/how-to/register.md:98]: Is `structure` also needed?

**Review by padraic-shafer — COMMENTED** [review]
Good catch. Here are a few other comments.

File: docs/source/how-to/register.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (danielballan) [reply docs/source/how-to/register.md:98]: It sure is!

**Review by genematx — APPROVED** [review]

**Pull request merged**